GDJS/Runtime/events-tools/commontools.ts

```diff
@@ -131,6 +131,21 @@ namespace gdjs {
         return !bool;
       };
 
+      /**
+       * Normalize a value between `min` and `max` to a value between 0 and 1.
+       * @param {number} val The value to normalize
+       * @param {number} min The minimum
+       * @param {number} max The maximum
+       * @returns The normalized value between 0 and 1
+       */
+      export const normalize = function (
+        val: float,
+        min: float,
+        max: float
+      ): number {
+        return (val - min) / (max - min);
```

Comment:
We should handle the case `max === min`. In this case, return `max`, as you've normalized a value to a range which is a single number, so the normalized number *is* this number.

Reply:
(same in C++ to avoid a crash)

Reply:
Done